packages/ai-code-mode-snippets/README.md

```
@@ -133,47 +133,47 @@ Options:
 
 // Node-only — `/storage` subpath
-import { createFileSkillStorage } from '@tanstack/ai-code-mode-skills/storage'
+import { createFileSnippetStorage } from '@tanstack/ai-code-mode-snippets/storage'
 ```
 
-#### `createFileSkillStorage(options)`
+#### `createFileSnippetStorage(options)`
 
 Git-friendly file-based storage:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)